app/components/markdown/markdown_table/markdown_table.js

```diff
@@ -67,21 +69,50 @@ export default class MarkdownTable extends React.PureComponent {
         });
     };
 
-    handleContentSizeChange = (contentWidth, contentHeight) => {
+    handleContentSizeChange = (_containerHeight, contentHeight) => {
```

Comment:
Isn't this first parameter still the width of the content? Also, we don't generally prefix variables with an underscore, so it would be nice if you could change this back

Reply:
Ah yes, my mistake with calling it `containerHeight` instead, I'll just revert that back to its original value